ui/components/src/ui-components/viewport/ViewportComponent.tsx

```diff
@@ -150,6 +153,23 @@ export class ViewportComponent extends React.Component<ViewportProps, ViewportSt
     }
   }
 
+  private static async cloneViewState(imodelConnection: IModelConnection, viewState: ViewState): Promise<ViewState | undefined> {
+    const className = viewState.classFullName;
```

Comment:
Why not `return viewState.clone();`?

Reply:
I didn't find viewState.clone()! Ugh.

Reply:
Yeah it's way down on `EntityState`. It's not `async` either so you can probably just delete your `cloneViewState` entirely.

Reply:
I did exactly that -- removed my cloneViewState() and returned viewState.clone().